### PR TITLE
Add Question results summary page

### PR DESCRIPTION
**Pull request by aitorlb**
Related issue: #57 

Go to http://localhost:3000/question_summaries/preview to compare the new view (left) to the normal question view (right).

Screenshot from 2021-03-21 21-19-47

Screenshot from 2021-03-21 21-17-23

### CODE REVIEW

**Comment by aitorlb** [comment]
Needs to be rebased after #62 is merged

**Comment by microstudi** [comment]
@aitorlb ready to rebase!

**Comment by microstudi** [comment]
Sorry @aitorlb this needs rebase again, cos' I've merged #63

**Review by microstudi — CHANGES_REQUESTED** [review]
Thanks @aitorlb this looks and behaves nice!

However we will need to work a little further in it:
- currently only the questions can be viewed in a summary mode.
- We should be able to do the same for a specific consultation so we can see all the questions and navigate the summary of the question when clicked.

Something like:
http://localhost:3000/consultations/laboriosam-voluptatem/summary
behaving similar as the question summary view, and when clicking on the question "view results" button direct the user to the question summary view.

We also going to need a "return to consultation" button on the top of the question summary.

With this we will be good to go.

File: app/views/decidim/consultations/consultation_summaries/_question.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <p><%= translated_attribute question.subtitle %></p>
+      </div>
+    </div>
+
+    <div class="columns mediumlarge-8 large-3 card--process__column">
+      <div class="card--full__image consultations-card__image"
+           style="background-image: url(<%= question.banner_image_url %>)">
+        <div class="card__content row collapse">
+          <div class="columns large-10 large-push-1">
+            <%= link_to "#{question_path(question)}/summary",
```
Comment (microstudi) [inline app/views/decidim/consultations/consultation_summaries/_question.html.erb:17]: This should be done by using Rails routes to take into account query variables (ie: locale)
```suggestion
      <%= link_to "#{Rails.application.routes.url_helpers.question_summary_path(question, locale: params[:locale])}",
```

**Review by microstudi — CHANGES_REQUESTED** [review]
I've been testing this, it is ok, 3 things:
- [ ] some old customizations collide with the new views (method `previous_published_questions` do not exist). This is due `NeedsQuestion` is overriden.
- [ ] `/summary` path must be done via Rails routes to take into account locales
- [ ] Readme needs a description of the feature

Note: I'll commit the fixes myself due the timing! Thanks for the work @aitorlb !

**Pull request merged**